concepts/80 Troubleshooting/30 3rd-Party Frameworks Issues/10 Fluent UI/05 Panels, Modals, Dialogs.md

```diff
@@ -0,0 +1,58 @@
+If you use controls like Fluent UI <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/panel" target="_blank">Panel</a>, <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/modal" target="_blank">Modal</a>, or <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog" target="_blank">Dialog</a> with DevExtreme components, you may experience issues listed below.
```

Comment:
```suggestion
If you use controls like Fluent UI <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/panel" target="_blank">Panel</a>, <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/modal" target="_blank">Modal</a>, or <a href="https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog" target="_blank">Dialog</a> with DevExtreme components, you may experience issues listed below.
```
How many issues? you may experience one of the issues listed below or several of the issues listed below?

Reply:
Several issues